pom.xml

```diff
@@ -110,6 +111,16 @@
             <artifactId>script-security</artifactId>
             <version>1.33</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>structs</artifactId>
+            <version>1.10</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>credentials</artifactId>
+            <version>2.1.14</version>
```

Comment:
[JENKINS-47498](https://issues.jenkins-ci.org/browse/JENKINS-47498) would avoid the need for this kind of change.